docs/source/extension/extension_dev.rst

```diff
@@ -1,12 +1,27 @@
 .. _developer_extensions:
 
-Extension Developer Guide
-=========================
+Develop Extensions
```

Comment:
The new naming using action verb is really good :+1:

Reply:
thanks :-)